packages/core/admin/package.json

```diff
@@ -34,7 +34,8 @@
     "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
     "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
     "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll"
+    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce:cov": "cross-env IS_EE=false jest --config ./jest.config.front.js --coverage --collectCoverageFrom='<rootDir>/packages/core/admin/admin/**/*.js' --coverageDirectory='<rootDir>/packages/core/admin/coverage'"
```

Comment:
Could you move that change into a seperate PR, so that others can have a look? I think we we introduce that, it would be nice to create some consistency between `CE` and `EE` and back-end and front-end ...

Reply:
Ok, sure I can...I have already added a similar script in a previous PR...you can find it here packages/core/helper-plugin/package.json